Automattic/wp-calypso
# Migration Flow Improvements: Adjust spacings in the left column

Related to #92468 

## Proposed Changes

* Increase margin and gap in the left column.
* Increase width of the right column.

## Why are these changes being made?

* After the changes in the right column its size increased, which made the left column look worse.

## Testing Instructions

* Open calypso.localhost:3000/start and continue till the Upgrade step (choose a free domain, check migration, enter a domain for migration, choose migrate the website and do it yourself).
* Check the spacings (margin in the left column and the gap between parts of the left column) increased, and there is no enormously large empty space in the bottom.

## Screenshots

Without Intro offer
![CleanShot 2024-07-07 at 23 45 26@2x](https://github.com/Automattic/wp-calypso/assets/329356/fc12226c-5bd3-4ba3-8426-ba7821173c30)

With Intro offer
![CleanShot 2024-07-08 at 00 04 43@2x](https://github.com/Automattic/wp-calypso/assets/329356/a96d0027-2ffd-4314-9d4a-24935945955f)

## Pre-merge Checklist
